torch/csrc/autograd/custom_function.h

```diff
@@ -13,6 +13,7 @@ namespace autograd {
 
 using optional_variable_list = std::vector<c10::optional<Variable>>;
 using _jvp_fn_t = std::function<variable_list(variable_list, variable_list)>;
+using _view_as_fn_t = std::function<at::Tensor(at::Tensor, at::Tensor)>;
```

Comment:
Why two arguments? You always call it with `(x, x)`

Reply:
haha good point

Reply:
updated to take a single argument